tt_eager/tt_dnn/op_library/bmm/multi_core_reuse_mcast_1d_optimized/bmm_op_multi_core_reuse_mcast_1d_optimized.cpp

```diff
@@ -834,6 +834,7 @@ operation::ProgramWithCallbacks create_program_mcast_in1(
     }
     if (in0_is_sharded) {
         mm_kernel_in0_sender_defines["IN0_SHARDED"] = "1";
+        mm_kernel_in0_sender_defines["SKIP_MCAST"] = "1";
```

Comment:
This is redundant? Line 841 adds it later.

Reply:
yep thx